api/spec/controllers/mno_enterprise/jpi/v1/marketplace_controller_spec.rb

```diff
@@ -117,89 +123,55 @@ def hash_for_apps(apps)
-      #
-      #   let!(:user) { build(:user) }
-      #   let!(:current_user_stub) { stub_user(user) }
-      #
-      #   before { sign_in user }
-      #   before { stub_api_v2(:get, "/organizations", [organization], [], { fields: 'id', filter: { 'users.id' => user.id }}) }
-      #   before { stub_api_v2(:get, '/apps', [app], [], { _metadata: { organization_id: organization.id } }) }
-      #   before do
-      #     stub_api_v2(:get, '/apps', [app], [],
-      #       { _metadata: { organization_id: organization.id }, fields: { apps: 'updated_at' }, page:{ number: 1, size: 1 }, sort: '-updated_at'})
-      #   end
-      #
-      #   it { is_expected.to be_success }
-      # end
+      context 'with organization_id' do
+        subject { get :index, organization_id: organization.id }
+        let!(:organization) { build(:organization) }
+        let!(:user) { build(:user) }
+        let!(:current_user_stub) { stub_user(user) }
+
+        before { sign_in user }
+        before { stub_api_v2(:get, "/organizations", [organization], [], { fields: { organizations: 'id' }, filter: { id: organization.id, 'users.id' => user.id }, page: { number: 1, size: 1 } }) }
+        before { stub_api_v2(:get, '/apps', [app], DEPENDENCIES, { _metadata: { organization_id: organization.id }, filter: { active: true } }) }
+        before do
+          stub_api_v2(:get, '/apps', [app], [],
+                      {
+                        _metadata: { organization_id: organization.id },
+                        fields: { apps: 'updated_at' },
+                        page: { number: 1, size: 1 },
+                        sort: '-updated_at'
+                      }
+          )
+        end
+
+        it { is_expected.to be_success }
+      end
 
       describe 'caching' do
         context 'on the first request' do
           it { is_expected.to have_http_status(:ok) }
-
           it 'sets the correct cache headers' do
             subject
             header = response.headers['Last-Modified']
-
             expect(header).to be_present
-
             # Parse and serialise to get correct format and avoid ms difference
             expect(Time.rfc822(header).in_time_zone.to_s).to eq(app.updated_at.to_s)
-
           end
         end
-
-        context 'on a subsequent request'  do
-
+        context 'on a subsequent request' do
           before do
             request.env['HTTP_IF_MODIFIED_SINCE'] = last_modified.rfc2822
           end
-
           context 'if it is not stale' do
             # Can't be based on the previous request due to parsing and rounding issues with ms
             let(:last_modified) { app.updated_at + 10.minutes }
             it { is_expected.to have_http_status(:not_modified) }
           end
-
           context 'if it is stale' do
             let(:last_modified) { app.updated_at - 10.minutes }
             it { is_expected.to have_http_status(:ok) }
           end
         end
       end
-
-      context 'without apps' do
```

Comment:
Why is this example removed?
It's quite important as it's a regression test. The bug appeared on v1 and was fixed but got reintroduced on v2 since the specs where skipped